examples/bing-maps/README.md

```diff
@@ -7,7 +7,7 @@ This is an example integrating deck.gl with Bing Maps using the deck.gl-bing-map
 To run this example, you need a [Bing Maps API key](https://www.microsoft.com/en-us/maps/create-a-bing-maps-key). You can either set an environment variable:
 
 ```bash
-export BingMapsAPIKey=<bing_maps_api_key>
+export VITE_BING_MAPS_API_KEY=<bing_maps_api_key>
```

Comment:
Why VITE_ (search replace casualty or intentional?)

Reply:
Intended, but not a strong preference. Vite will automatically load env variables prefixed with VITE_, and ignores those that aren't, so our examples can be more standalone by using this convention:

https://vitejs.dev/guide/env-and-mode

Otherwise I believe the examples may be implicitly depending on the top-level config here:

https://github.com/visgl/deck.gl-community/blob/9be4ebc9cde1d7d0dc85368085439340a897600f/examples/vite.config.local.mjs#L27-L32